### PR TITLE
[NOTASK]:Updated Read Me File

### PR DESCRIPTION
**Pull request by poulami-saha**
Let's update the Read me file for the final project submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,71 @@
...
 
 ## Design
 
 The challenge is part of [Frontend Mentor](https://www.frontendmentor.io/) premium version, hence we are not adding a link of the Design file here.
 
-## More details will be added in due course of time
-
-# Getting Started
-
-1. Clone the repo. From the terminal, run:
```
Comment (joanwolk) [inline README.md:17]: Did you consider moving the detailed information on git/GitHub to the bottom of the README (below the section on Contributing), or into a separate markdown file? There may be parts of that info that remain very helpful, particularly the information on the expectations for Pull Requests.

Comment (poulami-saha) [reply README.md:17]: Yes, you are right. Its very important to keep it, I am just working on the separate markdown file, which I will attach in the prerequisites

Comment (poulami-saha) [reply README.md:17]: its done now

File: README.md [hunk 2]
```diff
@@ -1,90 +1,72 @@
...
+- [Contributing](#contributing)
+
+## Introduction
+
 This is an open source project developed as part of the [Open Source Project](https://www.linkedin.com/posts/frontendqueens_opensource-collaboration-community-activity-7201980886232317952-VBld?utm_source=share&utm_medium=member_desktop) initiative by the [FrontEnd Queens](https://www.linkedin.com/company/frontendqueens/)
 
-## The Challenge
+## Challenge
 
 The challenge is to build out a multi-page website and get it looking as close to the design as possible.
```
Comment (FevenSeyfu) [inline README.md]: As a suggestion if we can also include the requirements for user interactivity for this challenge as well,
```suggestion
## Challenge
The challenge is to build out a multi-page website and get it looking as close to the design as possible.Your users should be able to:

- See the correct content for each team member on the About page when the + icon is clicked
- Receive an error message when the contact form is submitted if:
    - The Name, Email Address or Message fields are empty should show "This field is required"
    - The Email Address is not formatted correctly should show "Please use a valid email address"
- View the optimal layout for the interface depending on their device's screen size
- See hover and focus states for all interactive elements on the page

**Review by FevenSeyfu — COMMENTED** [review]
@poulami-saha  you have included most of the important sections but based on  frontend mentor challenges readme format these are my suggestions to include,

- Screenshot / Gif to show how the website functions
- acknowledgement- since we are not adding the attributions link to the footer we can add it here in the readme to acknowledge frontend mentors for the design and also frontend queens
- Authors - contributors social/ FM link(optional since there are a number of us)
- link to repo and live site
- Built with /technologies we have used 
- Continuous development - since we are currently building only the static we can list features to include in the future to make it more dynamic (option)

**Comment by poulami-saha** [comment]
> * acknowledgement- since we are not adding the attributions link to the footer we can add it here in the readme to acknowledge frontend mentors for the design and also frontend queens

Introduction is the acknowledgement to both FrontEnd Mentors and Frontend Queens

**Comment by poulami-saha** [comment]
> * Built with /technologies we have used

Prerequisites section has those details

**Comment by poulami-saha** [comment]
> * link to repo and live site

Features Section has this details

**Comment by poulami-saha** [comment]
> * Screenshot / Gif to show how the website functions

Still waiting for the project to complete & added the Collaborator details

File: README.md [hunk 3]
```diff
@@ -1,90 +1,89 @@
...
+4. [Feven Seyfu](https://github.com/FevenSeyfu)
+5. [Joan Wolkerstorfer](https://github.com/joanwolk)
+6. [Kezia Baidoo](https://github.com/keziabaidoo)
+7. [Sehaj Wadhwa](https://github.com/sehajwadhwa)
+8. [Uzma Khan](https://github.com/Uzmakh)
+9. [Poulami Saha](https://github.com/poulami-saha)
+
+### Contributing
+
+We welcome contributions! Please follow these steps to contribute:
```
Comment (joanwolk) [inline README.md:84]: I recommend a link or at least reference to the githubLearning document from here

**Review by joanwolk — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (joanwolk) [reply README.md:84]: Otherwise, looks good to me!

Comment (poulami-saha) [reply README.md:84]: Added now

**Review by joanwolk — APPROVED** [review]

**Pull request merged**